### PR TITLE
ci: Fixes and reduces nunit tests

### PR DESCRIPTION
**Pull request by dbrasseur-aneo**
The aggregation priority test is too big and doesn't test extc# explicitely, so it should be ignored. This also fixes a configuration of the UnifiedTestHelperBase

### CODE REVIEW

File: Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/AggregationPriority/AggregationPriorityTest.cs [hunk 1]
```diff
@@ -319,6 +319,7 @@ private async Task<IEnumerable<TaskRaw>> GetDistribution(int nRows)
   /// </summary>
   /// <param name="squareMatrixSize">The size of the square matrix.</param>
   [TestCase(20)]
+  [Ignore("Too big")]
```
Comment (aneojgurhem) [inline Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/AggregationPriority/AggregationPriorityTest.cs:322]: Is it possible to write a faster test that validates the same behavior ? That would be the best alternative

File: Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/UnitTestHelperBase.cs [hunk 2]
```diff
@@ -101,9 +101,8 @@ public void InitLogger()
                                         applicationNamespace,
                                         applicationService);
 
-    Props = new Properties(TaskOptions,
-                           Configuration.GetSection("Grpc")["EndPoint"],
-                           5001);
+    Props = new Properties(Configuration,
```
Comment (aneojgurhem) [inline Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/UnitTestHelperBase.cs:104]: This fix should be in another PR and why this constructor exists if we don't want it ?

File: Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/AggregationPriority/AggregationPriorityTest.cs (hunk 1 above)
Comment (dbrasseur-aneo) [reply Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/AggregationPriority/AggregationPriorityTest.cs:322]: I guess by reducing the matrix size from 20 to like 5 or something, but this tests core / and the queue, not what ExtC# does. SO I'm not sure it's relevant, especially for CI/CD

File: Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/UnitTestHelperBase.cs (hunk 2 above)
Comment (dbrasseur-aneo) [reply Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/UnitTestHelperBase.cs:104]: Historical reasons, we could remove it possibly but it may be used by the client iirc

**Review by aneojgurhem — APPROVED** [review]
